string[] vehicle_names
bool waitOnLastTask 
---
bool success